src/Aether/Devices/Sensors/Measurement.cs

```diff
@@ -25,6 +26,9 @@ namespace Aether.Devices.Sensors
         public Pressure BarometricPressure =>
             new Pressure(_value, (PressureUnit)_unit);
 
+        public  VolatileOrganicCompoundIndex Voc =>
+            new VolatileOrganicCompoundIndex(_value, (VolatileOrganicCompoundIndexUnit)_unit);
```

Comment:
Need to eventually create a more scalable solution to this, but for now, these bits will need to be modified to support the new unit.

https://github.com/scalablecory/aether/blob/4c4d8dd47f6ca4c0c508cd80c11371c5f930dda1/src/Aether/Themes/MultiLineTheme.cs#L129-L136

https://github.com/scalablecory/aether/blob/4c4d8dd47f6ca4c0c508cd80c11371c5f930dda1/src/Aether/Themes/MultiLineTheme.cs#L153-L160

Reply:
Done